envs/backend.env

```diff
@@ -12,6 +12,12 @@ COIN_NAME="GLM"
 COIN=GLM
 CHAIN_ID=1337
 
+CORS_ENABLED=true
+CORS_ORIGIN=*
```

Comment:
Let's add a note in README.md about this. So that deployer knows to customize this for prod.